templates/vue/src/components/Lightbox/media/MultiContentMedia/PageMenu/index.vue

```diff
@@ -98,11 +104,36 @@ export default {
         })
         .filter(row => !row.node.popup)
     },
-    lockRows() {
-      return this.node.typeData.lockRows
+    menuGroups() {
+      const menuGroups = []
+      const firstMenuGroup = []
+      this.rows.forEach(row => {
+        row.node.typeData.isSecondaryNode
+          ? menuGroups.push([row])
+          : firstMenuGroup.push(row)
+      })
+      menuGroups.unshift(firstMenuGroup)
+      return menuGroups
     },
-    disabledFrom() {
-      return this.rows.findIndex(row => !row.node.completed)
+    filteredMenuGroups() {

```

Comment:
Could you please add some comments before rows, menuGroups, and filteredMenuGroups to explain what the **_role_** (or _purpose_) of each of these are and what their data structure is? It would also be great if you could outline the data structure of these in this PR fo clarify. I am having trouble wrapping my head around all of these things that are being used and also some passed into the menu groups as props...